numpy/core/numeric.py

```diff
@@ -2145,8 +2145,12 @@ def fromfunction(function, shape, **kwargs):
         The function is called with N parameters, where N is the rank of
         `shape`.  Each parameter represents the coordinates of the array
         varying along a specific axis.  For example, if `shape`
-        were ``(2, 2)``, then the parameters in turn be (0, 0), (0, 1),
-        (1, 0), (1, 1).
+        were ``(2, 2)``, then the parameters would in turn be
```

Comment:
I'm not an English major, but I believe this to be correct usage whereas the previous wording was a bit sketchy. "In turn" generally means "as a consequence of" when used infix, but "one after the other when used postfix.

https://en.oxforddictionaries.com/definition/in_turn